README.md

```diff
@@ -132,17 +132,24 @@ vars:
     pendo_<default_source_table_name>_identifier: your_table_name 
 ```
 
-#### 🚨 Snowflake Users
+#### 🚨 Snowflake Users 🚨
 You may need to provide the case-sensitive spelling of your source tables that are also Snowflake reserved words.
 
 In this package, this would apply to the `GROUP` source. If you are receiving errors for this source, include the following in your `dbt_project.yml` file:
-
 ```yml
 vars:
-  pendo_group_identifier: '"Group"' # as an example, must include this quoting pattern and your exact casing
+  pendo_group_identifier: '"Group"' # as an example, must include this quoting pattern and adjust for your exact casing
+```
 
-quoting:
-  identifier: true
+**Note!** if you have sources defined in your project's yml, the above will not work. Instead you will need to add the following where your group table is defined in your yaml:
```

Comment:
Thinking about this a bit more -- maybe an example of what sources defined in your project's yml would mean? (and similarly for the source pacakge)

Reply:
I have adjusted.